### ISSUE TITLE
Crash when using set_font_data and get_available_chars on DynamicFont

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#46118]
**Godot version:**
Godot 3.2.4 rc 2

**OS/device including version:**
Ubuntu 20.04

**Issue description:**
When executing 
```
extends Node2D

var q_DynamicFont : DynamicFont = DynamicFont.new()

func _process(_delta : float) -> void:
	if randi() % 2 == 0:
		print("Executing DynamicFont::set_font_data")

		var p_object_0 = DynamicFontData.new()
		q_DynamicFont.set_font_data(p_object_0)

	if randi() % 2 == 0:
		print("Executing DynamicFont::get_available_chars")

		q_DynamicFont.get_available_chars()
```
Godot crashes and shows this backtrace
```
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x46210) [0x7fdc14687210] (??:0)
[2] /usr/bin/godot(FT_Get_First_Char+0x3d) [0x48408fc] (/home/rafal/Pulpit/godot3.2/thirdparty/freetype/src/base/ftobjs.c:3768 (discriminator 1))
[3] DynamicFontAtSize::get_available_chars() const (/home/rafal/Pulpit/godot3.2/scene/resources/dynamic_font.cpp:271)
[4] DynamicFont::get_available_chars() const (/home/rafal/Pulpit/godot3.2/scene/resources/dynamic_font.cpp:862)
[5] MethodBind0RC<String>::call(Object*, Variant const**, int, Variant::CallError&) (/home/rafal/Pulpit/godot3.2/./core/method_bind.gen.inc:593 (discriminator 4))
[6] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/home/rafal/Pulpit/godot3.2/core/object.cpp:919 (discriminator 1))
[7] Variant::call_ptr(StringName const&, Variant const**, int, Variant*, Variant::CallError&) (/home/rafal/Pulpit/godot3.2/core/variant_call.cpp:1129 (discriminator 1))
[8] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Variant::CallError&, GDScriptFunction::CallState*) (/home/rafal/Pulpit/godot3.2/modules/gdscript/gdscript_function.cpp:1089)
[9] GDScriptInstance::call_multilevel(StringName const&, Variant const**, int) (/home/rafal/Pulpit/godot3.2/modules/gdscript/gdscript.cpp:1254)
[10] Node::_notification(int) (/home/rafal/Pulpit/godot3.2/scene/main/node.cpp:58)
[11] Node::_notificationv(int, bool) (/home/rafal/Pulpit/godot3.2/./scene/main/node.h:46 (discriminator 14))
[12] CanvasItem::_notificationv(int, bool) (/home/rafal/Pulpit/godot3.2/./scene/2d/canvas_item.h:166 (discriminator 3))
[13] Node2D::_notificationv(int, bool) (/home/rafal/Pulpit/godot3.2/./scene/2d/node_2d.h:38 (discriminator 3))
[14] Object::notification(int, bool) (/home/rafal/Pulpit/godot3.2/core/object.cpp:931)
[15] SceneTree::_notify_group_pause(StringName const&, int) (/home/rafal/Pulpit/godot3.2/scene/main/scene_tree.cpp:988)
[16] SceneTree::idle(float) (/home/rafal/Pulpit/godot3.2/scene/main/scene_tree.cpp:528 (discriminator 3))
[17] Main::iteration() (/home/rafal/Pulpit/godot3.2/main/main.cpp:2113)
[18] OS_X11::run() (/home/rafal/Pulpit/godot3.2/platform/x11/os_x11.cpp:3634)
[19] /usr/bin/godot(main+0x125) [0x16e0bfb] (/home/rafal/Pulpit/godot3.2/platform/x11/godot_x11.cpp:57)
[20] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fdc146680b3] (??:0)
[21] /usr/bin/godot(_start+0x2e) [0x16e0a1e] (??:?)
```

**Comment by Vignesh1-art** [comment]
I copy pasted your code I get "Identifier ""DynamicFont not declared in the current scope "

**Comment by qarmin** [comment]
Are you using Godot 3 or Godot 4?
Dynamic font was renamed in Godot 4 to Font

**Comment by Vignesh1-art** [comment]
I'm using Godot 4. Sorry I'm new here

**Comment by akien-mga** [comment]
Fixed by #48911.
